src/schema/schema.graphql

```diff
@@ -485,6 +485,7 @@ interface MailItem {
 type ClientInfoAttributes {
 	zimbraWebClientLoginURL: String
 	zimbraWebClientLogoutURL: String
+	zimbraWebClientSkipLogoff: String
```

Comment:
should this be boolean?

Reply:
that is true for all boolean types, still we convert it to boolean so we can use strict type checking of graphql

Reply:
Done. Also updated in zm-x-web. see https://github.com/ZimbraOS/zm-x-web/pull/3829/commits/f6880a4eff6b16dc37f33448d1c2e3f00d646ab1